Dockerfile.deploy

```diff
@@ -55,7 +64,13 @@ RUN pip install --no-cache-dir --exists-action=w --no-deps -r requirements/syste
     && pip install --no-cache-dir --exists-action=w --no-deps -r requirements/prod_without_hash.txt\
     && pip install --no-cache-dir --exists-action=w --no-deps -e .
 
-RUN echo -e "from olympia.lib.settings_base import *\n\
+# Link /usr/sbin/uwsgi and /usr/bin/uwsgi to deal with migration from Centos -> Debian
+RUN ln -s /usr/bin/uwsgi /usr/sbin/uwsgi
+
+# Fix for https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=783738
+RUN ln -s /usr/lib/python2.7/plat-*/_sysconfigdata_nd.py /usr/lib/python2.7/
+
+RUN echo "from olympia.lib.settings_base import *\n\
```

Comment:
Do we need "echo -e" here? I thought without -e, it would print a literal '\n' and put that into the `settings_local.py` file.

Reply:
This appears to be a difference in shells for Centos and Debian. While testing I was seeing `-e` as an invalid/ignored flag. In testing with both:

Without `-e`

```
Step 19/23 : RUN echo "from olympia.lib.settings_base import *\nSTYLUS_BIN = 'node_modules/stylus/bin/stylus'\nLE
SS_BIN = 'node_modules/less/bin/lessc'\nCLEANCSS_BIN = 'node_modules/clean-css-cli/bin/cleancss'\nUGLIFY_BIN = 'n
ode_modules/uglify-js/bin/uglifyjs'\nFXA_CONFIG = {'default': {}, 'internal': {}}\n"> settings_local.py
 ---> Running in 307cd2b614b0
Removing intermediate container 307cd2b614b0
 ---> 6fab64623288
Step 20/23 : RUN cat settings_local.py
 ---> Running in b28f72c0d7cf
from olympia.lib.settings_base import *
STYLUS_BIN = 'node_modules/stylus/bin/stylus'
LESS_BIN = 'node_modules/less/bin/lessc'
CLEANCSS_BIN = 'node_modules/clean-css-cli/bin/cleancss'
UGLIFY_BIN = 'node_modules/uglify-js/bin/uglifyjs'
FXA_CONFIG = {'default': {}, 'internal': {}}
```

with `-e`
```
Step 19/23 : RUN echo -e "from olympia.lib.settings_base import *\nSTYLUS_BIN = 'node_modules/stylus/bin/stylus'\
nLESS_BIN = 'node_modules/less/bin/lessc'\nCLEANCSS_BIN = 'node_modules/clean-css-cli/bin/cleancss'\nUGLIFY_BIN =
 'node_modules/uglify-js/bin/uglifyjs'\nFXA_CONFIG = {'default': {}, 'internal': {}}\n"> settings_local.py
 ---> Running in 7e70730d528f
Removing intermediate container 7e70730d528f
 ---> bcc624958146
Step 20/23 : RUN cat settings_local.py
 ---> Running in a4d6abe04d84
-e from olympia.lib.settings_base import *
STYLUS_BIN = 'node_modules/stylus/bin/stylus'
LESS_BIN = 'node_modules/less/bin/lessc'
CLEANCSS_BIN = 'node_modules/clean-css-cli/bin/cleancss'
UGLIFY_BIN = 'node_modules/uglify-js/bin/uglifyjs'
FXA_CONFIG = {'default': {}, 'internal': {}}
```

With `-e` leaves a `-e` in the file.